applications/nrf_desktop/configuration/nrf52810dmouse_nrf52810/prj.conf

```diff
@@ -46,7 +46,13 @@ CONFIG_DESKTOP_BLE_BOND_ENABLE=y
 CONFIG_CAF_BLE_ADV=y
 CONFIG_CAF_BLE_ADV_DIRECT_ADV=n
 CONFIG_CAF_BLE_ADV_FAST_ADV=y
-CONFIG_CAF_BLE_ADV_SWIFT_PAIR=y
+
+# Configure advertised data
+CONFIG_DESKTOP_ADV_PROVIDER_UUID16_ALL=y
```

Comment:
try to hide things behind the application's kconfig option - these options will be always enabled

Reply:
Will set them as `default y` in application's Kconfig